gnovm/pkg/gnolang/helpers.go

```diff
@@ -26,6 +29,11 @@ func IsRealmPath(pkgPath string) bool {
 		!ReGnoRunPath.MatchString(pkgPath)
 }
 
+// IsPurePackagePath determines whether the given pkgpath is for a gno package.
+func IsPurePackagePath(pkgPath string) bool {
+	return strings.HasPrefix(pkgPath, PackagePathPrefix)
+}
+
```

Comment:
I would prefer this to be simply `!IsRealmPath`. This makes it so that a path is either a pure package or a realm; as it stands, an stdlib is neither a realm nor a pure package. (It is a pure package).

Reply:
b2e28fd

Reply:
It broke things like linter, i'm digging to make the pattern you suggest work, i pass the PR in draft until i have something ready to review, thanks for comment :heart: